core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.directive.ts

```diff
@@ -210,6 +216,10 @@ angular.module('oppia').directive('skillConceptCardEditor', [
               }
             };
           };
+
+          $scope.$on('$destroy', function() {
```

Comment:
Please add corresponding equivalent of this somewhere so that we do not forget this (while we open migration of directives to new contributors we can such equivalents there then)

Reply:
Hi @bansalnitish -- just to check, what do you mean by this comment? (What is "this", and where is the "somewhere" you are referring to?)

Reply:
@seanlip, Maybe @bansalnitish meant what is the equivalent of `$scope.$on('$destroy'` in angular. This entire `$scope.$on('$destroy'` can be replaced with the `ngOnDestroy` lifecycle hook and maybe he wants it to be added to the angular migration doc so that new contributors don't get stuck or confused.